swarm/runtime/stepwise/routing/navigator.py

```
@@ -0,0 +1,341 @@
+
+if TYPE_CHECKING:
+    from swarm.config.flow_registry import FlowDefinition, StepDefinition
+    from swarm.runtime.navigator_integration import NavigationOrchestrator
+    from swarm.runtime.router import FlowGraph
+    from swarm.runtime.types import RunSpec, RunState
+
+logger = logging.getLogger(__name__)
+
+
+def route_via_navigator(
+    step: "StepDefinition",
```

Ruff (0.14.10)

Remove quotes from type annotation

Remove quotes

(UP037)